<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFind</class>
 <widget class="QDialog" name="DialogFind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabFind">
      <attribute name="title">
       <string>Find</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>41</y>
         <width>271</width>
         <height>110</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>270</width>
         <height>110</height>
        </size>
       </property>
       <property name="title">
        <string>Direction</string>
       </property>
       <widget class="QRadioButton" name="radioButtonUp">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonDown">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Down</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_CaseSensitive">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Case sensitive</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>14</y>
         <width>24</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditFind">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>14</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>210</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonFindNext">
       <property name="geometry">
        <rect>
         <x>301</x>
         <y>11</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Find next</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonFindCancel">
       <property name="geometry">
        <rect>
         <x>301</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabReplace">
      <attribute name="title">
       <string>Replace</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Find content</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditFindContent">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditReplaceContent">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReplaceFindNext">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Find next</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReplace">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReplaceAll">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace all</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReplaceCancel">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_CaseSensitiveReplace">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
